<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work-Play Timer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listTodo"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="itemEntryTodo"/>
        </item>
        <item>
         <widget class="QPushButton" name="itemAddTodo">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Add Task</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLCDNumber" name="counterTimeLeftMinutesTens">
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="counterTimeLeftMinutesOnes">
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLCDNumber" name="counterTimeLeftSecondsTens">
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="counterTimeLeftSecondsOnes">
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelCurrentTask">
          <property name="text">
           <string>Current Task:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="buttonStartStop">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonComplete">
            <property name="text">
             <string>Complete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSkip">
            <property name="text">
             <string>Skip</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Work:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelWorkTime">
              <property name="text">
               <string>30 min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="sliderTimeDivision">
            <property name="maximum">
             <number>60</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Play:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelPlayTime">
              <property name="text">
               <string>30 min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
